openapi.yaml

```diff
@@ -504,7 +504,7 @@ definitions:
         items:
           type: string
           minItems: 1
-          pattern: '^[0-9a-fA-F]{64}$'
+          pattern: '(^[0-9a-fA-F]{64}|[0-9a-fA-F]{80}$)'
```

Comment:
```suggestion
          pattern: '^([0-9a-fA-F]{64}|[0-9a-fA-F]{80})$'
```

Reply:
thanks! fixed in a couple different spots.